benchmark/realnest/hep/answers/q06_1.csv

```diff
@@ -0,0 +1,76 @@
+x|y
+14.875|39
```

Comment:
Does this query complete on your laptop? It takes a while  on mine. Same with q06_2

Reply:
I only could get the result running its parts, otherwise it times out on my laptop